client/state/reader/follows/actions.js

```diff
@@ -23,6 +23,26 @@ import {
  */
 const debug = debugModule( 'calypso:redux:reader-follows' );
 
+export function follow( { url, blogId, feedId } ) {
```

Comment:
do you think we should maybe move off blogId and only use feedId/url for following/unfollowing?